### DISCUSSION TITLE
Front end job without cs degree

### DISCUSSION THREAD

**Discussion by missionfrontend** [community/community#146820] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to get it

**Comment by Lut-Lat-Aung** [comment]
Focus on building a strong portfolio showcasing real-world projects and modern web technologies (HTML, CSS, JavaScript, and frameworks like React, Angular and laravel). You can gain experience through freelancing, open-source contributions, and internships. Make connection and socialize with professionals, attend local meetups, and apply to companies open to non-traditional backgrounds. I hope you find your answers here and good luck on your career.

**Reply by missionfrontend** [reply]
Thank you

**Reply by missionfrontend** [reply]
How can i make connection any website

**Reply by Lut-Lat-Aung** [reply]
You can connect with people by talking with them or collaborating in projects and participate in discussions. e.g Github discussions.

This is a useful website.
You can start your career in web development at Odin Project site
https://www.theodinproject.com/

Odin Project Discord - https://discord.gg/fbFCkYabZB

**Reply by missionfrontend** [reply]
But i had learned html css and javascript

**Reply by missionfrontend** [reply]
And i a m creating Api based project

**Reply by missionfrontend** [reply]
i will start learning React

**Reply by Lut-Lat-Aung** [reply]
Then you should go to this link - https://www.w3schools.com/REACT/react_intro.asp
